ci-cd/github-actions.md

```diff
@@ -0,0 +1,218 @@
+---
+description: Configure GitHub Actions
+keywords: CI/CD, GitHub Actions,
+title: Configure GitHub Actions
+---
+
+This page guides you through the process of setting up a GitHub Action CI/CD pipeline with Docker containers. Before setting up a new pipeline, we recommend that you take a look at [Ben's blog](https://www.docker.com/blog/best-practices-for-using-docker-hub-for-ci-cd/){:target="_blank" rel="noopener" class="_"} on CI/CD best practices .
+
+This guide contains instructions on how to:
+
+1. Use a sample Docker project as an example to configure GitHub Actions
+2. Set up the GitHub Actions workflow
+3. Optimize your workflow to reduce the number of pull requests and the total build time, and finally,
+4. Push only specific versions to Docker Hub.
+
+## Set up a Docker project
+
+Let’s get started. This guide uses a simple Docker project as an example. The [SimpleWhaleDemo](https://github.com/usha-mandya/SimpleWhaleDemo){:target="_blank" rel="noopener" class="_"} repository contains an Ngnix alpine image. You can either clone this repository, or use your own Docker project.
+
+![SimpleWhaleDemo](images/simplewhaledemo.png){:width="500px"}
+
+Before we start, ensure you can access [Docker Hub](https://hub.docker.com/) from any workflows you create. To do this:
+
+1. Add your Docker ID and Personal Access Token (PAT) as secrets to GitHub.
+2. To create a new token, go to [Docker Hub Settings](https://hub.docker.com/settings/security) and then click **New Access Token**.
+3. Let’s call this token **simplewhaleci**.
+
+    ![New access token](images/github-access-token.png){:width="500px"}
+
+4. Now, add this token to your username as secrets into the GitHub secrets UI.
```

Comment:
```suggestion
4. Now, add this Personal Access Token (PAT) as a second secret into the GitHub secrets UI with the name `DOCKER_HUB_ACCESS_TOKEN`.
```

Reply:
I suggest to write the secret names in the docs to make it readable in addition to the screenshots.